### ISSUE TITLE
Filter by JSONB column

### ISSUE THREAD

**Issue by jcliftonlee** [SoftwareBrothers/adminjs#1123]
**Describe the problem feature solves**
We'd like to be able to query by nested values of JSONB columns.

**Describe the solution you'd like**
It'd be nice if AdminJS could detect JSONB columns and surface a form field for searching the contents.

**Comment by jcliftonlee** [comment]
@dziraf Makes sense. Might there potentially be any stop-gap solution, in the meantime? Like a hook or something where we could run some custom SQL or augment the TypeORM query object?

**Comment by dziraf** [comment]
You could technically create your own custom `list` handler: https://github.com/SoftwareBrothers/adminjs/blob/master/src/backend/actions/list/list-action.ts

`@adminjs/typeorm` has a custom filter parser which you can use to create any query:
https://github.com/SoftwareBrothers/adminjs-typeorm/blob/master/src/utils/filter/custom-filter.parser.ts

I think you can create a custom filter component as well in case you need to.

**Comment by dziraf** [comment]
Closing in favour of https://github.com/SoftwareBrothers/adminjs/issues/251
